frontend/app/configuration/Links.scala

```diff
@@ -12,7 +13,15 @@ object Links {
 
   val membershipFront = "http://www.theguardian.com/membership"
 
-  val membershipTerms = "http://www.theguardian.com/info/2014/sep/09/guardian-membership-terms-and-conditions"
+  def membershipTerms(countryGroup: Option[CountryGroup] = None) = {
+    countryGroup match {
+      case Some(cg) if cg.id.toLowerCase == "uk" => "https://www.theguardian.com/info/2014/sep/09/guardian-membership-terms-and-conditions"
```

Comment:
Can you compare a CountryGroup to a Country without assessing either the DefaultCountry or Countries props of the group?